### PR TITLE
Correcting badness around browser pro, lite, pro+spa agent types

### PR DESCRIPTION
**Pull request by zuluecho9**
Details in DOC-7085. Doing bulk of work related to this, a little more later. For explanation of the situation that is main root of why this needs changing, see the new section about 'Agent types' in the install doc. All the rest of the changes are related to that reality.

### CODE REVIEW

File: src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx [hunk 1]
```diff
@@ -16,19 +16,42 @@ redirects:
   - /docs/browser/new-relic-browser/installation/install-browser-monitoring-agent
 ---
 
-Browser uses a [JavaScript snippet](/docs/browser/new-relic-browser/page-load-timing/instrumentation-page-load-timing) (or "agent") to instrument your app's webpages. The JavaScript collects data for browser monitoring. To install the browser agent, you can choose from a number of deployment options.
+Browser uses a [JavaScript snippet](/docs/browser/new-relic-browser/page-load-timing/instrumentation-page-load-timing), also referred to as an "agent," to instrument your app's webpages. The JavaScript collects data for browser monitoring. To install the browser agent, you can choose from a number of deployment options.
```
Comment (urbiz-nr) [inline src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx:19]: Shouldn't it be "agent", ?

File: src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx [hunk 2]
```diff
@@ -16,19 +16,42 @@ redirects:
...
 
 <Callout variant="tip">
   To use browser and the rest of our [observability platform](https://one.newrelic.com), join the New Relic family! [Sign up](https://newrelic.com/signup) to create your free account in only a few seconds. Then ingest up to 100GB of data for free each month. Forever.
 </Callout>
 
-## Select a deployment option [#options]
+## Enable browser monitoring [#enable]
 
-No matter which option you use to deploy browser monitoring, the end result is to inject the browser agent's JavaScript snippet into your pages for browser monitoring. The method you select depends on your preferences and business needs. To view the procedure to install and enable the browser agent, click the link for the option you want to use. You can also use the [Browser **Application settings** page](/docs/browser/new-relic-browser/configuration/browser-settings-ui-options-apdex-geography) to update settings.
+To get started enabling browser: Go to **[one.newrelic.com](https://one.newrelic.com)**, select **Browser**, and then select **Add more data**. The UI gives you instructions.
```
Comment (urbiz-nr) [inline src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx:27]: "To get started enabling" reads a little long for me. How about a more direct "To enable"?

File: src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx [hunk 3]
```diff
@@ -16,19 +16,42 @@ redirects:
...
+To get started enabling browser: Go to **[one.newrelic.com](https://one.newrelic.com)**, select **Browser**, and then select **Add more data**. The UI gives you instructions. 
 
-<Callout variant="important">
-  The following configuration options refer only to the browser monitoring agent. These are not the same as the New Relic [user roles](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#roles) and [editions](https://newrelic.com/pricing). 
-</Callout>
+Some notes about this process: 
+* It can take several minutes after enabling the agent for data to show up in New Relic. 
+* If you are doing the copy/paste method, see [Copy/paste option](#copy-paste-app) for tips.
```
Comment (urbiz-nr) [inline src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx:31]: you're

File: src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx [hunk 4]
```diff
@@ -87,65 +110,30 @@ No matter which option you use to deploy browser monitoring, the end result is t
...
 
 ## Instrument webpages using the APM agent [#agent-instrumentation]
 
 This information applies to apps that are also monitored by APM.
 
-New Relic's agents can instrument webpages with the required JavaScript for page load timing. If you use the APM agent to add the JavaScript snippet to your webpages, insert the instrumentation snippet as close to the top as possible. This allows you to take advantage of detailed information about browser's [AJAX calls](/docs/browser/new-relic-browser/dashboard-details/ajax-dashboard) and [JavaScript errors](/docs/browser/new-relic-browser/dashboard-details/javascript-errors-dashboard).
+New Relic's agents can instrument webpages with the required JavaScript for page load timing. If you are using an APM agent's API to add the JavaScript snippet to your webpages, insert the instrumentation snippet as close to the top as possible. This allows you to take advantage of detailed information about browser's [AJAX calls](/docs/browser/new-relic-browser/dashboard-details/ajax-dashboard) and [JavaScript errors](/docs/browser/new-relic-browser/dashboard-details/javascript-errors-dashboard).
```
Comment (urbiz-nr) [inline src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx:136]: you're

File: src/content/docs/browser/new-relic-browser/configuration/browser-app-settings-page.mdx [hunk 5]
```diff
@@ -0,0 +1,15 @@
+---
+title: 'Configure browser agent data collection with the app settings page'
+metaDescription: "For New Relic browser monitoring: use the app settings page to configure the agent type and deployment method."
+---
+
+There are several configuration options that regular what data the browser agent reports. This is a factor when [enabling browser monitoring](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent), and you can also change these configurations afterward.
```
Comment (urbiz-nr) [inline src/content/docs/browser/new-relic-browser/configuration/browser-app-settings-page.mdx:6]: regular? Shouldn't it be regulate?

File: src/content/docs/browser/single-page-app-monitoring/get-started/install-single-page-app-monitoring.mdx [hunk 6]
```diff
@@ -11,44 +11,18 @@ redirects:
...
 
-In the New Relic UI, you can enable [single-page app (SPA) monitoring](/docs/browser/single-page-app-monitoring/get-started/introduction-single-page-app-monitoring).
+Single page app (SPA) monitoring comes with the default [install of the browser agent](/docs/browser/single-page-app-monitoring/get-started/spa-compatibility-requirements/). 
 
 <Callout variant="tip">
   To use SPA and browser monitoring, as well as the rest of our [observability platform](https://one.newrelic.com), join the New Relic family! [Sign up](https://newrelic.com/signup) to create your free account in only a few seconds. Then ingest up to 100GB of data for free each month. Forever.
 </Callout>
 
-## Prerequisites [#prerequites]
+## Requirements
```
Comment (urbiz-nr) [inline src/content/docs/browser/single-page-app-monitoring/get-started/install-single-page-app-monitoring.mdx:20]: Add title link.

**Review by urbiz-nr — APPROVED** [review]
This was very interesting to peer edit. And long overdue!

File: src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx (hunk 1 above)
Comment (zuluecho9) [reply src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx:19]: that's how most of the world does it, but U.S. is dumb in that regard. Personally I do it the way you do and the way the rest of the world does it.

**Pull request merged**